File: codegen/objc/objc-entities/src/main/groovy/com/stanfy/helium/handler/codegen/objectivec/entity/file/ObjCStringSourcePart.java
package com.stanfy.helium.handler.codegen.objectivec.entity.file;

/**
 * Created by ptaykalo on 9/10/14.
 * Simplies objc Source part, which  simply returns contents
 */
public class ObjCStringSourcePart implements ObjCSourcePart {

  private String contents = "";

  public ObjCStringSourcePart(final String contents) {
    this.contents = contents;
  }

  @Override
  public String asString() {
    return contents;
  }
}

File: codegen/objc/objc-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/objectivec/entity/file/ObjCStringSourcePart.kt
package com.stanfy.helium.handler.codegen.objectivec.entity.file;

/**
 * Created by ptaykalo on 9/10/14.
 * Simpliest objc Source part, which  simply returns contents
 */
public class ObjCStringSourcePart(val contents: String = "") : ObjCSourcePart {

  override fun asString(): String {
    return contents;
  }

}
